libcomposefs/lcfs-internal.h

```diff
@@ -21,6 +21,8 @@
 #include "lcfs-fsverity.h"
 #include "hash.h"
 
+#include <endian.h>
```

Comment:
Minor nit: I think it's best practice to include system headers before internal ones.

Reply:
i followed what all the other files already do, which do after instead of before (well, lcfs-mount.c does both)

Reply:
Yes, you're right; sorry.  (A classic "small PR is much more likely to get bikeshed changes")